plone/app/controlpanel/search.py

```diff
@@ -19,13 +19,11 @@
 from widgets import MultiCheckBoxThreeColumnWidget as MCBThreeColumnWidget
 
 
-anon_auth_items = (('anon', _(u'anonymous users')),
-                   ('auth', _(u'logged in users'),))
+SortSearchResultsVocabulary = SimpleVocabulary(
+    [SimpleTerm(value=u'relevance', title=_(u'relevance')),
+     SimpleTerm(value=u'Date', title=_(u'date (newest first)')),
+     SimpleTerm(value=u'sortable_title', title=_(u'alphabetically'))])
 
-anon_auth_terms = [SimpleTerm(item[0], title=item[1]) for item in
-                   anon_auth_items]
-
-AnonAuthVocabulary = SimpleVocabulary(anon_auth_terms)
```

Comment:
This old vocabulary is not used anywhere (I search into .py and .pt files into omelette)